index.html

```diff
@@ -1,82 +1,101 @@
-            <p>
-                Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
-            </p>
-        </div>
+            <nav>
+                <ul>
+                    <li>
+                        <a href="#search-engine-optimization">Search Engine Optimization</a>
+                    </li>
+                    <li>
+                        <a href="#online-reputation-management">Online Reputation Management</a>
+                    </li>
+                    <li>
+                        <a href="#social-media-marketing">Social Media Marketing</a>
+                    </li>
+                </ul>
+            </nav>
+    </HEADER>
+
+
+    <div class="hero">
+
     </div>
-    <div class="benefits">
+
+
+    <main>
+        <section id="search-engine-optimization" class="service">
+                <img src="./assets/images/search-engine-optimization.jpg" alt="Image of notebook with SEO on the cover on a desk with pencils." class="float-left" />
+                <h2>Search Engine Optimization</h2>
+                <p>
+                    The dominance of mobile internet use means that users are searching for the right business as they travel, shop, or sit on their couch at home. Search Engine Optimization (SEO) allows you to increase your visibility and find the right customers for your business.
+                </p>
+        </section>
+        <section id="online-reputation-management" class="service">
+                <img src="./assets/images/online-reputation-management.jpg" alt="Image of person speaking on mobile phone with laptop showing a rising chart for online reputation." class="float-right" />
+                <h2>Online Reputation Management</h2>
+                <p>
+                    The web is full of opinions, and some of these can be negative. Social media allows anyone with an internet connection to say whatever they want about your business. Online Reputation Management gives you the control over what potential customers see when they search for your business.
+                </p>
+        </section>
+        <section id="social-media-marketing" class="service">
+                <img src="./assets/images/social-media-marketing.jpg" alt="Image of table top with a bunch of random social media icons and actions .Also people are sat arund the table" class="float-left" />
+                <h2>Social Media Marketing</h2>
+                <p>
+                    Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
+                </p>
+        </section>
+    </main>
+      
+    
+
+
+    <aside class="benefits">
         <div class="benefit-lead">
```

Comment:
```suggestion
        <div class="benefit">
```
If you notice in the stylesheet there is duplication of the styles applied for the 3 benefits. They could all be classed a single style declaration with a class name like `benefit` or any name of your choosing. Code duplication is not preferred and for this task it is important to consider minimising obvious duplication of code. Please fix this and also make the relevant changes in the style sheet.

Reply:
done